templates/Layout/Page.ss

```diff
@@ -9,7 +9,7 @@
     </div>
     <div class="row">
         <section class="col-lg-8<% if not $Children %> offset-lg-2<% end_if %>">
-            <% if $ElementalArea %>
+            <% if $ElementalAreaID %>
```

Comment:
How did this even work before?

Watea would the same issue https://github.com/silverstripe/cwp-watea-theme/blob/3/templates/Layout/HomePage.ss

Majority of Starter/Watea installs would not have Elemental, so this is pretty odd